packages/scope-manager/tools/generate-lib.ts

```diff
@@ -1,5 +1,6 @@
 import type { TSESTree } from '@typescript-eslint/types';
 import { AST_TOKEN_TYPES } from '@typescript-eslint/types';
+import { ESLint } from '@typescript-eslint/utils/ts-eslint';
```

Comment:
@bradzacher this seem to be generating some issues on my machine

```bash
> yarn build

NX   Could not execute command because the task graph has a circular dependency
rule-schema-to-typescript-types:build --> type-utils:build --> utils:build --> scope-manager:build --> utils:build
```